environment

```diff
@@ -95,6 +95,7 @@ DSS_CERTIFICATE_VALIDATION="EMAIL"
 DSS_ZONE_NAME="dev.data.humancellatlas.org."
 PYTHONWARNINGS=ignore:ResourceWarning,ignore::UserWarning:zipfile:
 DSS_SECRETS_STORE="dcp/dss"
+DSS_PARAMETER_STORE="dcp/dss"
```

Comment:
README for ENV values needs to be updated as well.

Reply:
fixed